libs/src/services/dataContracts/AudiusContracts.ts

```diff
@@ -128,17 +165,28 @@ class AudiusContracts {
         this.getRegistryAddressForContract,
         this.logger
       )
+      // @ts-expect-error -- TODO: this looks like a missed bug
```

Comment:
what's the bug here and is it worth flagging in Slack?

Reply:
yeah that's a good idea! the bug is that UserReplicaSetManagerClient.init doesn't take any args. so not user if we can just remove selectNewEndpointOnRetry, or build that into the init

Reply:
ahh I see. it looks like it's always false anyway. wdyt about removing it?

Reply:
yeah given that, def will do!